app/Http/Livewire/Sidebar.php

```diff
@@ -73,15 +71,13 @@ public function generateAlbumStructure()
 		$share = new \stdClass();
 		$share->title = Lang::get('ALBUM_SHARING');
 		$_public = $this->album->is_public ? Lang::get('ALBUM_SHR_YES') : Lang::get('ALBUM_SHR_NO');
-		$_hidden = $this->album->requires_link ? Lang::get('ALBUM_SHR_YES') : Lang::get('ALBUM_SHR_NO'); // TODO : double check;
-		$_downloadable = $this->album->is_downloadable ? Lang::get('ALBUM_SHR_YES') : Lang::get('ALBUM_SHR_NO');
-		$_share_button_visible = $this->album->is_share_button_visible ? Lang::get('ALBUM_SHR_YES') : Lang::get('ALBUM_SHR_NO');
+		$_hidden = $this->album->is_link_required ? Lang::get('ALBUM_SHR_YES') : Lang::get('ALBUM_SHR_NO'); // TODO : double check;
```

Comment:
What is the TODO-comment supposed to tell me?

Reply:
Was a note for me, but I am rewriting pretty much the full thing in Livewire so does not matter.